docs/content/preview/releases/release-notes/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: YugabyteDB releases
+headerTitle: YugabyteDB releases
+linkTitle: Releases
+description: An overview of the preview and current stable YugabyteDB database releases, and upcoming releases.
+image: /images/section_icons/index/quick_start.png
+type: indexpage
+showRightNav: true
+cascade:
+  unversioned: true
+---
+
+## Releases
+
+| Release series | Released | End of maintenance support | End of Life (EOL) |
+| :------------- | :------- | :------------------------- | :---------------- |
+| [v2.19](v2.19/) <span class='metadata-tag-gray'>Preview</span> | {{< yb-eol-dates "v2.19" release >}} | n/a | n/a |
+| [v2.18](v2.18/) <span class='metadata-tag-green'>STS</span> | {{< yb-eol-dates "v2.18" release >}} | {{< yb-eol-dates "v2.18" EOM >}} | {{< yb-eol-dates "v2.18" EOL >}} |
+| [v2.16](v2.16/) <span class='metadata-tag-green'>STS</span> | {{< yb-eol-dates "v2.16" release >}} | {{< yb-eol-dates "v2.16" EOM >}} | {{< yb-eol-dates "v2.16" EOL >}} |
+| [v2.14](v2.14/) <span class='metadata-tag-green'>LTS</span> | {{< yb-eol-dates "v2.14" release >}} | {{< yb-eol-dates "v2.14" EOM >}} | {{< yb-eol-dates "v2.14" EOL >}} |
+
+{{< tip title="YugabyteDB Anywhere release notes have moved" >}}
+Starting with v2.16, the [release notes for YugabyteDB Anywhere](../yba-releases/) have moved to their own page.
+{{< /tip >}}
+
+## Releases at end of life (EOL) {#eol-releases}
+
+{{<note title="Archived docs available">}}
+Documentation for EOL stable releases is available at the [YugabyteDB docs archive](https://docs-archive.yugabyte.com/).
+{{</note>}}
+
+The following releases are no longer supported:
+
+| Release series | Released | End of maintenance support | End of Life (EOL) |
+| :------------- | :------- | :------------------------- | :---------------- |
+| [v2.17](v2.17/) | December 8, 2022 | n/a | n/a |
+| [v2.15](v2.15/) | June 27, 2022 | n/a | n/a |
+| [v2.13](end-of-life/v2.13/) | March 7, 2022 | n/a | n/a |
+| [v2.12](end-of-life/v2.12/) | {{< yb-eol-dates "v2.12" release >}} | {{< yb-eol-dates "v2.12" EOM >}} | {{< yb-eol-dates "v2.12" EOL >}} |
+| [v2.11](end-of-life/v2.11/) | November 22, 2021 | n/a | n/a |
+| [v2.9](end-of-life/v2.9/) | August 31, 2021 | n/a | n/a |
+| [v2.8](end-of-life/v2.8/) | November 15, 2021 | November 15, 2022 | May 15, 2023 |
+| [v2.7](end-of-life/v2.7/) | May 5, 2021 | n/a | n/a |
+| [v2.6](end-of-life/v2.6/) | July 5, 2021 | July 5, 2022 | January 5, 2023 |
+| [v2.5](end-of-life/v2.5/) | November 12, 2020 | n/a | n/a |
+| [v2.4](end-of-life/v2.4/) | January 22, 2021 | January 22, 2022 | July 21, 2022 |
+| [v2.2](end-of-life/v2.2/) | July 15, 2020 | July 15, 2021 | January 15, 2022 |
+| [v2.1](end-of-life/v2.1/) | February 25, 2020 | February 25, 2021 | August 08, 2021 |
+| [v2.0](end-of-life/v2.0/) | September 17, 2019 | September 17, 2020 | March 03, 2021 |
+| [v1.3](end-of-life/v1.3/) | July 15, 2019 | July 15, 2020 | January 15, 2021 |
+
+All non-current preview releases are unsupported.
```

Comment:
Reading this line gave me a thought: hey wait, what does this mean . What kinda releases are they referring to in the above table? And so my suggestion to add "Preview" to "release series".